### PR TITLE
Update `state` and `import` commands

### PR DESCRIPTION
**Pull request by euliancom**
## Description
- updated the `state` command
- added new option `batch` for `import` command 
- Update convert to HCL issue #876 

## Type of Changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] This change requires a change to the documentation

### `default.tfimport` file
```
landing_zone_route_pcx,aws_route.landing_zone_route_pcx_mitocgroup,rtb-123qwee123123q12w_172.31.0.0/16,aws.mitocgroup
landing_zone_route_pcx,aws_route.landing_zone_route_pcx_terrahub,rtb-123qwee123123q12w_172.31.0.0/16,aws.terrahub
landing_zone_route_pcx,aws_route.landing_zone_route_pcx_terrahub,rtb-123qwee123123q12w_172.31.0.0/16,aws.terrahub
landing_zone_route_pcx,aws_route.landing_zone_route_pcx_terrahub,rtb-123qwee123123q12w_172.31.0.0/16,aws.terrahub
landing_zone_route_pcx,aws_route.landing_zone_route_pcx_terrahub,rtb-123qwee123123q12w_172.31.0.0/16,aws.terrahub
landing_zone_route_pcx_old,aws_route.landing_zone_route_pcx_terrahub,rtb-123qwee123123q12w_172.31.0.0/16,aws.terrahub
```

### run command `import`
```
root@DESKTOP-ECSQPB1:/mnt/c/Terrahub/terraform-aws-landing-zone# terrahub import -b ../default.tfimport -i landing_zone_route_pcx
💡 THUB_TOKEN is not provided.
💡 [landing_zone_route_pcx] terraform init -no-color -force-copy -input=false .
[landing_zone_route_pcx]
Initializing the backend...
[landing_zone_route_pcx]
[landing_zone_route_pcx] Initializing provider plugins...
[landing_zone_route_pcx]
[landing_zone_route_pcx] The following providers do not have any version constraints in configuration,
so the latest version was installed.

To prevent automatic upgrades to new major versions that may contain breaking
[landing_zone_route_pcx] changes, it is recommended to add version = "..." constraints to the
corresponding provider blocks in configuration, with the constraint strings
suggested below.

* provider.aws: version = "~> 2.32"

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
[landing_zone_route_pcx] rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.
💡 [landing_zone_route_pcx] terraform workspace list
[landing_zone_route_pcx] * default

💡 [landing_zone_route_pcx] terraform import -no-color -provider=aws.mitocgroup -var-file='/root/.terrahub/cache/hcl/landing_zone_route_pcx_eef16dcf/default.tfvars' -input=false aws_route.landing_zone_route_pcx_mitocgroup rtb-123qwee123123q12w_172.31.0.0/16
[landing_zone_route_pcx] aws_route.landing_zone_route_pcx_mitocgroup: Importing from ID "rtb-123qwee123123q12w_172.31.0.0/16"...
[landing_zone_route_pcx] aws_route.landing_zone_route_pcx_mitocgroup: Import prepared!
[landing_zone_route_pcx]   Prepared aws_route for import
aws_route.landing_zone_route_pcx_mitocgroup: Refreshing state... [id=r-rtb-123qwee123123q12w3854007479]
[landing_zone_route_pcx]
Import successful!
[landing_zone_route_pcx]
The resources that were imported are shown above. These resources are now in
your Terraform state and will henceforth be managed by Terraform.

💡 [landing_zone_route_pcx] terraform import -no-color -provider=aws.terrahub -var-file='/root/.terrahub/cache/hcl/landing_zone_route_pcx_eef16dcf/default.tfvars' -input=false aws_route.landing_zone_route_pcx_terrahub[0] rtb-123qwee123123q12w_172.31.0.0/16
[landing_zone_route_pcx] aws_route.landing_zone_route_pcx_terrahub[0]: Importing from ID "rtb-123qwee123123q12w_172.31.0.0/16"...
[landing_zone_route_pcx] aws_route.landing_zone_route_pcx_terrahub[0]: Import prepared!
  Prepared aws_route for import
aws_route.landing_zone_route_pcx_terrahub[0]: Refreshing state... [id=r-rtb-123qwee123123q12w3854007479]
[landing_zone_route_pcx]
Import successful!
[landing_zone_route_pcx]
The resources that were imported are shown above. These resources are now in
your Terraform state and will henceforth be managed by Terraform.

💡 [landing_zone_route_pcx] terraform import -no-color -provider=aws.terrahub -var-file='/root/.terrahub/cache/hcl/landing_zone_route_pcx_eef16dcf/default.tfvars' -input=false aws_route.landing_zone_route_pcx_terrahub[1] rtb-123qwee123123q12w_172.31.0.0/16
[landing_zone_route_pcx] aws_route.landing_zone_route_pcx_terrahub[1]: Importing from ID "rtb-123qwee123123q12w_172.31.0.0/16"...
[landing_zone_route_pcx] aws_route.landing_zone_route_pcx_terrahub[1]: Import prepared!
  Prepared aws_route for import
aws_route.landing_zone_route_pcx_terrahub[1]: Refreshing state... [id=r-rtb-123qwee123123q12w3854007479]
[landing_zone_route_pcx]
[landing_zone_route_pcx] Import successful!

The resources that were imported are shown above. These resources are now in
[landing_zone_route_pcx] your Terraform state and will henceforth be managed by Terraform.

💡 [landing_zone_route_pcx] terraform import -no-color -provider=aws.terrahub -var-file='/root/.terrahub/cache/hcl/landing_zone_route_pcx_eef16dcf/default.tfvars' -input=false aws_route.landing_zone_route_pcx_terrahub[2] rtb-123qwee123123q12w_172.31.0.0/16
[landing_zone_route_pcx] aws_route.landing_zone_route_pcx_terrahub[2]: Importing from ID "rtb-123qwee123123q12w_172.31.0.0/16"...
[landing_zone_route_pcx] aws_route.landing_zone_route_pcx_terrahub[2]: Import prepared!
[landing_zone_route_pcx]   Prepared aws_route for import
aws_route.landing_zone_route_pcx_terrahub[2]: Refreshing state... [id=r-rtb-123qwee123123q12w]
[landing_zone_route_pcx]
Import successful!
[landing_zone_route_pcx]
The resources that were imported are shown above. These resources are now in
your Terraform state and will henceforth be managed by Terraform.

💡 [landing_zone_route_pcx] terraform import -no-color -provider=aws.terrahub -var-file='/root/.terrahub/cache/hcl/landing_zone_route_pcx_eef16dcf/default.tfvars' -input=false aws_route.landing_zone_route_pcx_terrahub[3] rtb-0cac2bfe68a72f262_172.31.0.0/16
[landing_zone_route_pcx] aws_route.landing_zone_route_pcx_terrahub[3]: Importing from ID "rtb-0cac2bfe68a72f262_172.31.0.0/16"...
[landing_zone_route_pcx] aws_route.landing_zone_route_pcx_terrahub[3]: Import prepared!
  Prepared aws_route for import
aws_route.landing_zone_route_pcx_terrahub[3]: Refreshing state... [id=r-rtb-0cac2bfe68a72f2623854007479]
[landing_zone_route_pcx]
[landing_zone_route_pcx] Import successful!

The resources that were imported are shown above. These resources are now in
[landing_zone_route_pcx] your Terraform state and will henceforth be managed by Terraform.

✅ Done
```

### Test Configuration
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

## Checklist:
- [x] I have read the **CONTRIBUTING** document.
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have checked that my changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/commands/import.js [hunk 1]
```diff
@@ -22,21 +25,58 @@ class ImportCommand extends TerraformCommand {
...
+      return Promise.all(
+        configContentArr.map(it => {
+          const resourceData = it.split('=');
+          linesMap.push({
+            fullAddress: resourceData[0],
+            value: resourceData[1],
+            provider: providerContent
+          })
+          return linesMap;
+        }))
```
Comment (aluchianic) [inline src/commands/import.js]: Promise.all is redundant - no async operations

File: src/commands/import.js [hunk 2]
```diff
@@ -22,21 +25,54 @@ class ImportCommand extends TerraformCommand {
   run() {
     const configContentArr = this.getOption('config');
     const providerContent = this.getOption('provider');
+    const batch = this.getOption('batch');
+    const include = this.getOption('include');
```
Comment (eistrati) [inline src/commands/import.js]: please rethink this functionality to reuse `--include`, `--exclude`, `--include-regex` and `--exclude-regex` defined in `terraform-command.js` (ref: https://github.com/TerraHubCorp/terrahub/blob/dev/src/terraform-command.js#L43)

Comment (eistrati) [reply src/commands/import.js]: will be addressed in future iteration of this feature

File: src/commands/import.js [hunk 3]
```diff
@@ -22,21 +26,71 @@ class ImportCommand extends TerraformCommand {
...
+      let linesMap = [];
       configContentArr.map(it => {
         const resourceData = it.split('=');
+        linesMap.push({
+          component: '',
+          fullAddress: resourceData[0],
+          value: resourceData[1],
+          provider: (providerContent !== '') ? `-provider=${providerContent}` : '',
+          overwrite: overwrite,
+          hash: config[Object.keys(config)[0]].project.code
```
Comment (aluchianic) [inline src/commands/import.js]: can be simplified `Object.values(config)[0].project.code)`

**Pull request merged**